docs/antctl.md

```diff
@@ -33,6 +33,7 @@ running in three different modes:
   - [Flow Aggregator commands](#flow-aggregator-commands)
     - [Dumping flow records](#dumping-flow-records)
     - [Record metrics](#record-metrics)
+  - [Multi-Cluster commands](#multi-cluster-commands)
```

Comment:
I think we generally use "Multi-cluster"?

Reply:
done

Reply:
I changed it back to `Multi-Cluster` since this is automatically generated, and it will fail during TOC CI verify if we change it to `Multi-cluster`

Reply:
ignore me, I shouldn't change here, I updated `Multi-Cluster` to `Multi-cluster` in the description section, then we can have the right TOC title here.